src/Simplex/Chat/Store/Groups.hs

```diff
@@ -1224,6 +1230,34 @@ updateGroupMemberAccepted db User {userId} m@GroupMember {groupMemberId} status
     (status, role, currentTs, userId, groupMemberId)
   pure m {memberStatus = status, memberRole = role, updatedAt = currentTs}
 
+updateGroupMembersRequireAttention :: DB.Connection -> User -> GroupInfo -> GroupMember -> GroupMember -> IO GroupInfo
+updateGroupMembersRequireAttention db user@User {userId} g@GroupInfo {groupId, membersRequireAttention} member member'
+  | gmRequiresAttention member' && not (gmRequiresAttention member) =
+      increaseGroupMembersRequireAttention db user g
+  | not (gmRequiresAttention member') && gmRequiresAttention member = do
```

Comment:
better put them into names

Reply:
gmRequiresAttention can be made to return 1/0, then it's just 

| was == now -> pure g
| otherwise -> add (now - was)